packages/flutter/test/material/input_chip_test.dart

```diff
@@ -370,32 +368,37 @@ void main() {
   });
 
   testWidgets('Material3 - Input chip has correct selected color when enabled', (WidgetTester tester) async {
-    final ChipThemeData material3ChipDefaults = ThemeData(useMaterial3: true).chipTheme;
+    final ThemeData theme = ThemeData(useMaterial3: true);
+    final ChipThemeData material3ChipDefaults = theme.chipTheme;
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(enabled: true),
-      useMaterial3: true,
+      theme: theme,
     );
 
     final RenderBox materialBox = getMaterialBox(tester);
     expect(materialBox, paints..rrect(color: material3ChipDefaults.backgroundColor));
   });
 
   testWidgets('Material3 - Input chip has correct selected color when disabled', (WidgetTester tester) async {
-    final ChipThemeData material3ChipDefaults = ThemeData(useMaterial3: true).chipTheme;
+    final ThemeData theme = ThemeData(useMaterial3: true);
+    final ChipThemeData material3ChipDefaults = theme.chipTheme;
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(),
-      useMaterial3: true,
+      theme: theme,
     );
 
     final RenderBox materialBox = getMaterialBox(tester);
     expect(materialBox, paints..path(color: material3ChipDefaults.disabledColor));
```

Comment:
```suggestion
    expect(materialBox, paints..path(color: material3ChipDefaults.disabledColor));
```

same here

Reply:
Fixed in https://github.com/flutter/flutter/commit/1ab12f11b973923bbc0cb050f09bf639da3856f6